# <Insert Fun Name Here>

## Rust style exception handling in TypeScript